ta_services/ta_services.mk

```diff
@@ -0,0 +1,15 @@
+ta_dirs := keymaster
```

Comment:
Since `CFG_TA_SERVICES_KEYMASTER` is introduced, it should either be used or removed.

This doesn't seem to deal with 32/64-bit compilation. I don't think it makes sense to compile a specific TA for both, but it should be configurable. Perhaps `CFG_TA_SERVICES_KEYMASTER` could carry that information?

Reply:
@jenswi-linaro Maybe let's introduce additional var (I think it's better to keep CFG_TA_SERVICES_KEYMASTER as bool, to avoid confustions in future + no need to specify arch for each TA in `ta_services`, it's redundant), that will be used will configure 32/64-bit compilation for all TAs in `ta_services`

Reply:
I don't think we can rule out that someone will want to mix 32-bit and 64-bit TAs on a 64-bit platform.

Reply:
@jenswi-linaro Ok, so let's introduce tristate vars here (`n`, `64`, `32`)

Reply:
@jenswi-linaro please check current solution, I hope I understood your suggestion right